containers/Containerfile

```diff
@@ -0,0 +1,91 @@
+FROM registry.fedoraproject.org/fedora-toolbox:34
+
+MAINTAINER "Candlepin Team" <candlepin@redhat.com>
+
+ENV SMDEV_CONTAINER_OFF='True'
+
+RUN yum install --disablerepo=updates -y \
+    bind-utils \
+    cmake \
+    dbus-daemon \
+    dbus-glib \
+    dbus-glib-devel \
+    desktop-file-utils \
+    dnf-utils \
+    gcc \
+    GConf2-devel \
+    gettext \
+    git \
+    git \
+    glibc-common \
+    gtk3-devel \
```

Comment:
I hope that we don't need this gkt3 thing anymore!

Reply:
I don't think we do, will update!